history-of-the-quakers-v2/original/05-book-3-chapter-11.adoc

```diff
@@ -302,28 +326,34 @@ because Christ Jesus hath said I shall not swear at all.
 If I might gain the whole world for swearing an oath I could not,
 and whatever I have to lose this day for not swearing an oath I offer it up.
 
-Judge.
+[.discourse-part]
+_Judge._
 What say you to the indictment?
 
-M+++.+++ Fell.
+[.discourse-part]
+_M+++.+++ Fell._
 What should I say?
 I am clear and innocent of wronging any man upon
 the earth as the little child that stands by me;
 and if any here have any thing to lay to my charge,
 let them come and testify it before you all;
 and if I be clear and innocent you have no law against me.
 
+[.offset]
```

Comment:
So throughout all this discourse (and Howgill's below too), [.offset] seems to be needed to break up their words from the interjections of narration, but they make some really big spaces if they are before [.embedded-content-document.].

Reply:
oops sorry, not [.embedded-content-document]....I meant [.discourse-part]

Reply:
i think these look good, i can fix the spacing when they are next to embedded docs 👍